.github/actions/restore-accounts/action.yml

```
@@ -0,0 +1,32 @@
+name: Restore accounts
+description: Restore accounts to be consumed by Playwright
+inputs:
+  ACCOUNTS:
+    description: "Accounts encrypted string"
+    required: true
+  E2E_ENCODE_PASS:
+    description: "Password for encoding credentials"
+    required: true
+  BASE_URL:
+    description: "Dashboard base url"
+    required: true
+  API_URL:
+    description: "API url"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version-file: ".nvmrc"
+
+    - name: Restore accounts
+      shell: bash
+      env:
+        E2E_ENCODE_PASS: ${{ inputs.E2E_ENCODE_PASS }}
+        ACCOUNTS: ${{ inputs.ACCOUNTS }}
+        API_URL: ${{ inputs.API_URL }}
+        BASE_URL: ${{ inputs.BASE_URL }}
+      run: |
+        node playwright/auth.js restore $ACCOUNTS
```

It should be quoted to prevent globbing and word splitting:
```suggestion
        node playwright/auth.js restore "$ACCOUNTS"
```